### PR TITLE
Add cacert rule for java 8

### PR DESCRIPTION
**Pull request by bgzzz**
We need to add cacert for java truststore 

It is done a little bit different for different Java versions.

This commit adds support of cacert setup for Java 8

### CODE REVIEW

**Review by sorenmat — COMMENTED** [review]
I feel like some documentation needs updating :)

File: src/maven.ts [hunk 1]
```diff
@@ -65,18 +66,27 @@ export async function setupMaven(opts: MavenOpts): Promise<void> {
     `-Djavax.net.ssl.keyStore=${p12Path} -Djavax.net.ssl.keyStoreType=pkcs12 -Djavax.net.ssl.keyStorePassword=${opts.password}`
   );
 
+  var params: string[] = ['-importcert'];
```
Comment (fardin01) [inline src/maven.ts:69]: Please consider changing `-importcert` to `-importcacert` just so it won't be confused with a client cert.

File: src/maven.ts [hunk 2]
```diff
@@ -65,18 +66,27 @@ export async function setupMaven(opts: MavenOpts): Promise<void> {
...
+    await exec.exec(
+      path.join(opts.javaPath, 'bin/keytool'),
+      params.concat([
+        '-storepass',
+        'changeit',
+        '-noprompt',
+        '-alias',
+        'mycert',
+        '-file',
+        rooCaPath
```
Comment (fardin01) [inline src/maven.ts:87]: Missing a t in `rooCaPath`

**Review by fardin01 — APPROVED** [review]
🎖️  Left my review.

File: src/maven.ts (hunk 1 above)
Comment (bgzzz) [reply src/maven.ts:69]: these are only allowed options 
```
-importcert         Imports a certificate or a certificate chain
 -importpass         Imports a password
 -importkeystore     Imports one or all entries from another keystore

**Comment by bgzzz** [comment]
> I feel like some documentation needs updating :)

https://github.com/Tradeshift/tradeshift-puppet/blob/d8e7e6f89e4ba0b085c3fc578910604ad7ee2e5a/scripts/employee-jre/install-tradeshift-ca.sh#L55

**Comment by sorenmat** [comment]
Perhaps add a CODEOWNERS file while in there :)

**Review by blacksails — APPROVED** [review]

**Pull request merged**